src/amo/components/SearchResults.js

```diff
@@ -72,18 +47,16 @@ class SearchResults extends React.Component {
       }
     }
 
-    const message = (
-      (<p ref={(ref) => { this.message = ref; }} className={classNames({
-        'visually-hidden': hideMessageText,
-        'SearchResults-message': !hideMessageText,
-      })}>{messageText}</p>)
-    );
-
     return (
       <div ref={(ref) => { this.container = ref; }} className="SearchResults">
-        {resultHeader}
-        {message}
-        {searchResults}
+        <AddonCard addons={results}>
```

Comment:
It doesn't render that text as part of the header/footer, but instead as the extra content inside the `AddonsCard` (that prefixes the `addons.map()` output). The content inside an `AddonsCard` is put before the rendered addons.